File: SQL/Midterm files/Acme_Used_CarsWPFcs/Acme_Used_CarsWPFcs/Acme_Used_CarsDB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Data;
using System.Data.SqlClient;

namespace Acme_Used_CarsWPFcs
{
    public static class Acme_Used_CarsDB
    {
        public static SqlConnection GetConnection()
        {
            SqlConnection connection = new SqlConnection();
            connection.ConnectionString = "Data Source=localhost\\sqlexp2012;Initial Catalog='Acme Used Cars';Integrated Security=SSPI";
            return connection;
        }
    }
}

File: SQL/Midterm files/Acme_Used_CarsWPFvb/Acme_Used_CarsWPFvb/Acme_Used_CarsDB.vb
Imports System.Data
Imports System.Data.SqlClient

Public Class Acme_Used_CarsDB

    Public Shared Function GetConnection() As SqlConnection
        Dim con As New SqlConnection()
        con.ConnectionString = "Data Source=localhost\sqlexp2012;Initial Catalog='Acme Used Cars';Integrated Security=SSPI"
        Return con
    End Function

End Class
